### DISCUSSION TITLE
访问网页时间断性地出现 SSL Version Error | SSL Version Error Appears intermittently when visiting web pages

### DISCUSSION THREAD

**Discussion by Cyb3rN8** [apernet/hysteria#501] [Q&A]
使用Hysteria访问Https网站时，会偶发性、间接性地出现 SSL 错误（似乎是在SSL握手阶段时中断而导致）

Occasional, indirect SSL errors when accessing Https sites using Hysteria (seems to be caused by interruptions during the SSL handshake phase)

**Comment by tobyxdd** [comment]
Log?

**Reply by Cyb3rN8** [reply]
I looked at another project, TUIC, which says that if you use 0-RTT, all first-time SSL sites will have errors, and I wonder if that's the reason

**Reply by haruue** [reply]
Hysteria does not use 0-RTT for the TCP traffic it carries. 
Whatever, could you please provide the link to any related discussion (or issue, doc, etc) you found?

**Reply by Cyb3rN8** [reply]
https://blog.cloudflare.com/even-faster-connection-establishment-with-quic-0-rtt-resumption/
https://www.ssltrust.com.au/blog/err_ssl_protocol_error
https://www.thesslstore.com/blog/fix-err-ssl-protocol-error/

**Reply by haruue** [reply]
@nathan-deng-ai 
你使用的是 TUN 或者 TProxy 作为 Hysteria 的入站吗？（例如路由器翻墙， Passwall）
要不试试禁用浏览器的 QUIC （ chrome://flags/#enable-quic ） 或者通过 SwitchyOmega 这样的插件使用 Hysteria 的 HTTP/socks5 入站， 看看还会出现这种问题吗？
反正 Hysteria 也没法加速它承载的 QUIC 流量。
